src/cts/src/TritonCTS.tcl

```diff
@@ -85,12 +85,12 @@ sta::define_cmd_args "clock_tree_synthesis" {[-wire_unit unit]
                                              [-num_static_layers] \
                                              [-sink_clustering_buffer] \
                                              [-obstruction_aware] \
-					     [-apply_ndr] \
+					                                   [-apply_ndr] \
                                              [-insertion_delay] \
                                              [-sink_buffer_max_cap_derate] \
                                              [-use_dummy_load] \
                                              [-delay_buffer_derate] \
-                                            }
+                                            };# checker off
```

Comment:
why off?

Reply:
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/39641663/98371a91-011e-4569-b7a1-360cb29f3eef)
Obsolete flag